src/python/py/models/builder.py

```diff
@@ -2491,6 +2653,11 @@ def create_model(model_name, input_path, output_dir, precision, execution_provid
             onnx_model = Phi3Mini4KModel(config, io_dtype, precision, execution_provider, cache_dir, extra_options)
         elif config.architectures[0] == "Phi3ForCausalLM" and config.max_position_embeddings == 131072:
             onnx_model = Phi3Mini128KModel(config, io_dtype, precision, execution_provider, cache_dir, extra_options)
+        elif config.architectures[0] == "PhiMoEForCausalLM" and config.max_position_embeddings == 131072:
```

Comment:
The `config.max_position_embeddings == 131072` check exists for Phi-3 mini to distinguish between the 4K and 128K variants. Since Phi-3.5 MoE has only one variant, I think we can remove that part of the check.

Reply:
the team shared the 4K version to us before. just want to make it clear that it's the 128k version.